roles/taca/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+- name: Fetch TACA-ngi-pipeline from GitHub
+  git: repo={{ taca_ngi_repo }}
+       dest={{ taca_ngi_dest }}
+       version={{ taca_ngi_version }}
+       force=yes
+  tags: taca
+
+- name: Install TACA-ngi-pipeline requirements
+  shell: /lupus/ngi//sw/anaconda/envs/NGI/bin/pip install -r /lupus/ngi/sw/taca-ngi-pipeline/requirements.txt
+  tags: taca
+
+- name: Install TACA-ngi-pipeline
+  shell: cd {{ taca_ngi_dest }} && /lupus/ngi/sw/anaconda/envs/NGI/bin/pip install .
+  tags: taca
+
+-name: Create TACA delivery configs directory
+  file: path="{{ ngi_resources }}/TACA" state=directory mode=g+s
+  tags: taca
+
+- name: Deploy sthlm_taca_delivery.yml config
```

Comment:
Instead of having two different copies of all the templates, where we in one copy use the Uppsala variables, and in the other copy use the Stockholm varibles, I suggest an other approach that we're using in the `ngi_pipeline` and `func_account` roles: 

- have a generic template, with a generic variable name 
- have site specific variables under the `defaults/main.yml` for Uppsala and Stockholm
- before the `template` task, use a `set_fact` task, that sets the generic variable in the template file to the site specific variable that you defined under `defaults` 

That way it gets a bit cleaner and organized -- with just one set of files, but different with variables passed into them at different times.